.github/CODEOWNERS

```diff
@@ -1 +1,43 @@
-.github/workflows @SamMousa @brunobowden
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence. 
+#
+# Security sensitive items must be at the end.
+#
+# All user entries should be alpha-sorted and lowercase.
+
+# These owners will be the default owners for everything in
+# the repo, unless a later match takes precedence.
+# Sadly, we have to repeat the top-level owners everywhere,
```

Comment:
Not sure if this really "sadly". We get to save the config (i.e. top-level owners) in the code as opposed to GitHub's UI.

Reply:
Good point.  Its still sad I can't define a macro though.